assets/src/pages/room/hooks/useMembraneClient.tsx

```diff
@@ -127,9 +135,7 @@ export const useMembraneClient = (
         setWebrtc(webrtc);
       })
       .receive("error", (response) => {
-        setErrorMessage("Connecting error");
-        console.error("Received error status");
-        console.error(response);
+        handleError(`Connecting error occurred`);
```

Comment:
Would be nice to give a little bit more descriptive error. E.g. `Couldn't establish signaling connection" or "Failed to establish WS connection"